tests/unit/assessment/test_pipelines.py

```diff
@@ -49,9 +117,32 @@ def test_pipeline_assessment_without_config(mocker):
             state=PipelineState.IDLE,
         )
     ]
-    ws = Mock()
+    ws = create_autospec(WorkspaceClient)
     config_dict = {}
+    pipeline_cluster = [
+        PipelineCluster(
+            apply_policy_default_values=None,
+            autoscale=None,
+            aws_attributes=None,
+            azure_attributes=None,
+            cluster_log_conf=None,
+            custom_tags={'cluster_type': 'default'},
+            driver_instance_pool_id=None,
+            driver_node_type_id=None,
+            gcp_attributes=None,
+            init_scripts=[],
+            instance_pool_id=None,
+            label='default',
+            node_type_id='Standard_F4s',
+            num_workers=1,
+            policy_id=None,
+            spark_conf=None,
+            spark_env_vars=None,
+            ssh_public_keys=None,
+        )
+    ]
     ws.pipelines.get().spec.configuration = config_dict
+    ws.pipelines.get().spec.clusters = pipeline_cluster
     crawler = PipelinesCrawler(ws, MockBackend(), "ucx")._assess_pipelines(sample_pipelines)
```

Comment:
don't invoke private methods in unit tests! it's prohibited.

Reply:
Removed the private method from unit test